pinot-plugins/pinot-stream-ingestion/pinot-pulsar/src/main/java/org/apache/pinot/plugin/stream/pulsar/PulsarPartitionLevelConsumer.java

```diff
@@ -45,21 +45,21 @@
 public class PulsarPartitionLevelConsumer extends PulsarPartitionLevelConnectionHandler
     implements PartitionGroupConsumer {
   private static final Logger LOGGER = LoggerFactory.getLogger(PulsarPartitionLevelConsumer.class);
-  private final ExecutorService _executorService;
+
   private final Reader _reader;
-  private boolean _enableKeyValueStitch;
 
-  public PulsarPartitionLevelConsumer(String clientId, StreamConfig streamConfig,
-      PartitionGroupConsumptionStatus partitionGroupConsumptionStatus) {
+  // TODO: Revisit the logic of using a separate executor to manage the request timeout. Currently it is not thread safe
```

Comment:
Is enableKeyValueStitch not used any more? 

@KKcorps - Could you verify as well?

Reply:
It is handled within `PulsarUtils.buildPulsarStreamMessage()`